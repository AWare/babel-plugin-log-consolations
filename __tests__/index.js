/* eslint-env jest */

import { transform } from 'babel-core'
import plugin from '../src'

var example = `
console.log('hello')
console.info('hello')
console.debug('hello')
console.warn('hello')
console.error('hello')
console.log('hello',1,2,'hi')
`

it('works', () => {
  const {code} = transform(example, {plugins: [plugin], filename: 'foo.js'})
  expect(code).toMatchSnapshot()
})
