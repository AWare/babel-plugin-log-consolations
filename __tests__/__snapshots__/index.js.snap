// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
"'use strict';

console.log('\\\\x1B[32m foo.js:2 \\\\x1B[0m', 'hello');
console.info('\\\\x1B[32m foo.js:3 \\\\x1B[0m', 'hello');
console.debug('\\\\x1B[32m foo.js:4 \\\\x1B[0m', 'hello');
console.warn('\\\\x1B[32m foo.js:5 \\\\x1B[0m', 'hello');
console.error('\\\\x1B[32m foo.js:6 \\\\x1B[0m', 'hello');
console.log('\\\\x1B[32m foo.js:7 \\\\x1B[0m', 'hello', 1, 2, 'hi');"
`;
